src/Regridder.jl

```diff
@@ -633,5 +633,52 @@ function cgll2latlonz(field; DIR = "cgll2latlonz_dir", nlat = 360, nlon = 720, c
     return new_data, coords
 end
 
+"""
+    truncate_dataset(date0, timeStart, timeEnd, datafile)
+
+Truncates given data set, and constructs a new dataset containing only the dates that are used in the simulation
+"""
+function truncate_dataset(date0, time_start, time_end, datafile, datapath)
+    date_start = date0 + Dates.Second(time_start)
+    date_end = date0 + Dates.Second(time_start + time_end)
+
+    ds = NCDataset(datafile, "r")
+    dates = ds["time"][:]
+
+    if date_start < dates[1]        # if the simulation start date is before our first date in the dataset, we leave the beginning of the truncated dataset to be the first date available
```

Comment:
Could you add the comments in their own line above the if/else statements please? It's nice to keep the line length < ~80 characters when we can